src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -367,6 +367,9 @@
 
   </Target>
 
+  <!-- This target get all project items that may need to be transferred to the output directory for the up to date check. -->
```

Comment:
Why does this target exist?

There's an existing target that already produces these items: https://github.com/dotnet/msbuild/blob/c10f34c4e00644e06120cb87f323c330f29a1e89/src/Tasks/Microsoft.Common.CurrentVersion.targets#L4732.

Reply:
Do you mean that `GetCopyToOutputDirectoryItems `can return items that doesn't exist which will make FUTD to incorrectly validate a project is up-to-date ?

Reply:
Let's step back, what is this target producing?

Reply:
this line was removed. 
This line was causing to override CopyToOutputDirectoryItem in design-time that is/was (don't know which executes first) also generated in MSBuild.

Reply:
removed